libraries/chain/include/eosio/chain/chain_config.hpp

```diff
@@ -322,6 +387,41 @@ inline DataStream &operator<<(DataStream &s, const eosio::chain::data_entry<eosi
    return s;
 }
 
+/**
+ * @brief This is for packing data_entry<chain_config_v2, ...>
+ * that is used as part of packing data_range<chain_config_v2, ...>
+ * @param s datastream
+ * @param entry contains config reference and particular id
+ * @throws unsupported_feature if protocol feature for particular id is not activated
+ */
+template <typename DataStream>
+inline DataStream &operator<<(DataStream &s, const eosio::chain::data_entry<eosio::chain::chain_config_v2, eosio::chain::config_entry_validator> &entry){
+   using namespace eosio::chain;
+
+   //initial requirements were to skip packing field if it is not activated.
+   //this approach allows to spam this function with big buffer so changing this behavior
+   //moreover:
+   //The contract has no way to know that the value was skipped and is likely to behave incorrectly.
+   //When the protocol feature is not activated, the old version of nodeos that doesn't know about
+   //the entry MUST behave the same as the new version of nodeos that does.
+   //Skipping known but unactivated entries violates this.
+   EOS_ASSERT(entry.is_allowed(), unsupported_feature, "config id ${id} is no allowed", ("id", entry.id));
```

Comment:
"not allowed"

Reply:
https://github.com/AntelopeIO/spring/pull/1257/commits/c77065ab4cd07db17cdd94598170cc1cfc8caea5
I also corrected "no allowed" in all other places in the same file.